tools/SetupFlow/DevHome.SetupFlow/ViewModels/LoadingViewModel.cs

```diff
@@ -180,7 +179,8 @@ public void AddMessage(string message, MessageSeverityKind severityKind = Messag
     {
         Application.Current.GetService<WindowEx>().DispatcherQueue.TryEnqueue(() =>
         {
-            var messageToDisplay = new LoadingMessageViewModel(message);
+            var messageToDisplay = _host.GetService<LoadingMessageViewModel>();

```

Comment:
Both `Application.Current` and `host` do the same thing at runtime.

The difference is in unit tests.  `Application.Current` throws an exception because `Application.Current` does not exist.  

Always go with `host` so unit tests won't crash.